install/apache-flink/install.yml

```diff
@@ -0,0 +1,15 @@
+id: apache-flink
+name: Apache Flink
+title: Apache Flink monitoring
+description: |
+  New Relic’s instant observability quickstart with Apache Flink monitoring helps to detect client, jobmanager and taskmanager logs and parses them into a JSON payload.
+
+target:
+  type: integration
+  destination: host
+  os:
+    - linux
+install:
+  mode: link
+  destination:
+    url: https://docs.newrelic.com/
```

Comment:
We would suggest having this url direct to a specific doc for this install. If a user tries to access the docs to install this quickstart, they would not have any useful information

Reply:
Hi @akristen , request you to kindly provide us the Apache Flink Integration docs URL so that we can update it here.
Thank you. 😊